google/cloud/spanner/integration_tests/client_integration_test.cc

```diff
@@ -473,7 +473,7 @@ TEST_F(ClientIntegrationTest, CommitAtLeastOnceBatched) {
     ASSERT_THAT(commit_result, IsOk());
     EXPECT_THAT(commit_result->indexes, Not(IsEmpty()));
     for (auto index : commit_result->indexes) {
-      ASSERT_THAT(index, AllOf(Ge(0), Lt(groups.size())));
+      ASSERT_THAT(index, AllOf(Ge(0U), Lt(groups.size())));
```

Comment:
Q.  What build highlighted that?  I must have missed it.

In any case, given that `index` is unsigned, we can just omit the `>= 0` altogether.

Reply:
You did not miss it.  The Windows build after the `vcpkg` update started failing.  My PR did not include your changes, your PR did not include the `vcpkg` changes.

Reply:
Done.